components/file_attachment_list/index.ts

```diff
@@ -46,6 +48,7 @@ function makeMapStateToProps() {
             fileInfos,
             fileCount,
             isEmbedVisible: isEmbedVisible(state, ownProps.post.id),
+            pluginFileAttachmentComponents: state.plugins.components.FileAttachmentPluginComponent as unknown as FileAttachmentPluginComponent[],
```

Comment:
Can we have this strongly typed in the store [here](https://github.com/mattermost/mattermost-webapp/blob/d2cd25fc96b30f5098cd3cdd23d27d553fdfb166/types/store/plugins.ts#L40) rather than using `as unknown` here?